deepchem/feat/one_hot.py

```diff
@@ -8,52 +8,62 @@
 ]
 
 
-class OneHotFeaturizer(Featurizer):
-  """
-  NOTE(LESWING) Not Thread Safe in initialization of charset
+class OneHotFeaturizer(MolecularFeaturizer):
+  """Encodes a molecule as a one-hot array.
+
+  This featurizer takes a molecule and encodes its Smiles string as a one-hot
+  array.
+
+  Note
+  ----
+  This class requires RDKit to be installed. Note that this featurizer is not
+  Thread Safe in initialization of charset
   """
 
   def __init__(self, charset=None, padlength=120):
-    """
+    """Initialize featurizer.
+
     Parameters
     ----------
     charset: obj:`list` of obj:`str`
```

Comment:
I couldn't understand this params meaning...
If you can specify, maybe it is better to specify

Reply:
Ah good catch. Cleaned this up.